src/myrtlespeech/data/sampler.py

```diff
@@ -23,10 +46,47 @@ def _batch_indices(self, indices, batch_size, drop_last):
         return batches
 
     def __iter__(self):
-        if self.shuffle:
-            random.shuffle(self.batch_indices)
+        if self._n_iterators in self._sequential:
+            iter_ = self._seq_iter()
+        else:
+            iter_ = self._rnd_iter()
+        self._n_iterators += 1
+        return iter_
+
+    def _seq_iter(self):
         for b in self.batch_indices:
             yield b
 
+    def _rnd_iter(self):
+        indices = list(range(len(self.batch_indices)))
+        if self.shuffle:
+            random.shuffle(indices)
+        for index in indices:
+            yield self.batch_indices[index]
+
     def __len__(self):
         return len(self.batch_indices)
+
+
+class SortaGrad(SequentialRandomSampler):
+    """An iterable over batch indices according to the SortaGrad strategy.
+    The SortaGrad curriculum learning strategy iterates over batches from the
```

Comment:
Add newline. See docstring comment above.

Ref sortagrad paper?

Reply:
Done it! I think SortaGrad was first used in DeepSpeech2, so I've added a link to that paper